### DISCUSSION TITLE
Strimzi Topic Operator max number of managed topics

### DISCUSSION THREAD

**Discussion by iagotomas** [strimzi/strimzi-kafka-operator#9165] [Q&A]
Hi,
wanted to ask if anyone could tell about reaching the limits of the Topic Operator. How many topics can the topic operator handle with default resources values? Any advice scaling the operator? We're managing ~500 topics without hassle so far but we're increasingly growing that number so wanted to learn from others experience :)

**Comment by scholzj** [comment]
@tombentley Any idea?

**Reply by tombentley** [reply]
I don't have numbers, sorry. I think users have reported varying experiences with the bidirectional operator. 

One of the changes introduced in the new unidirectional topic is batched reconciliation. That should allow to support larger numbers of topics (for the same CPU resource), but I'm not aware that anyone has really tested the limits in terms of #topics, even with the default resources. So if you tested out the UTO I'd be interested to hear how you got on.

**Reply by iagotomas** [reply]
Thanks @tombentley , we are planning to test UTO, so far we're using the bidirectional operator. For now the ~500 topics are being managed without hassle. Will try to share our experience with UTO once we start testing it.
